chains/testnet/chain-config.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
This is the chain-specific info used to build the chain-spec. I try to limit the data that needs to be here, e.g. by pulling the actual token config from `compound-chain/networks/ropsten.json` where possible